src/portals/hope/features/claims/claim-details/ClaimSubclaims/components/AddSubclaim.tsx

```diff
@@ -95,25 +95,30 @@ gql`
   }
 `
 
-export const AddSubclaim: React.FC<{ claimId: string; memberId: string }> = ({
-  claimId,
-  memberId,
-}) => {
+export const AddSubclaim: React.FC<{
+  claimId: string
+  memberId: string
+}> = ({ claimId, memberId }) => {
   const [addSubclaim] = useAddSubclaimMutation()
-  const { data: claimTypesData } = useGetClaimTypesQuery()
   const { data: contractsData } = useMemberContractsQuery({
     variables: { memberId },
   })
 
   const contracts = contractsData?.member?.contracts ?? []
-  const types = claimTypesData?.claimTypes ?? []
 
   const [dropDownActive, setDropDownActive] = useState(false)
   const [selectedContract, setSelectedContract] =
     useState<MemberContractInfoFragment>()
   const [selectedType, setSelectedType] = useState('')
   const [newSubclaimStep, setNewSubclaimStep] = useState(1)
 
+  const selectedContractType = selectedContract?.currentAgreement.typeOfContract
```

Comment:
The ClaimTypeBinding uses `insuranceType` and not the `typeOfContract` (at least for now)
```suggestion
  const selectedInsuranceType = selectedContract?.insuranceType
```